pkg/logs/docker/docker.go

```diff
@@ -16,6 +16,7 @@ import (
 // See https://godoc.org/github.com/moby/moby/client#Client.ContainerLogs:
 // [8]byte{STREAM_TYPE, 0, 0, 0, SIZE1, SIZE2, SIZE3, SIZE4}[]byte{OUTPUT}
 const messageHeaderLength = 8
+const maxDockerBufferSize = 16 * 1024
```

Comment:
Can we add a comment and a link to the docker repository, where the corresponding constant is defined please ?